packages/aws-cdk-lib/rosetta/aws_docdb/default.ts-fixture

```diff
@@ -1,5 +1,5 @@
 // Fixture with packages imported, but nothing else
-import { Stack } from 'aws-cdk-lib';
+import { RemovalPolicy, Stack } from 'aws-cdk-lib';
```

Comment:
Curious why is this change needed?

Reply:
I've added an example using `RemovalPolicy.SNAPSHOT` in the DocDB README file.
It's required for rosetta to resolve the dependency.